db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_suspend_vm_ok.yaml

```diff
@@ -0,0 +1,16 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name:
+    name: USER_SUSPEND_VM_OK
+    inherits:
+    description:
+  fields:
+  - rel4:
+      value: "/System/event_handlers/event_action_refresh?target=src_vm"
+  - rel5:
+      value: "/System/event_handlers/change_event_target_state?target=src_vm&param=suspended"
```

Comment:
Minor, but perhaps put this first and then refresh?  Or, just do it in the same order we do it for vmware.